packages/alfa-rules/src/sia-r11/rule.ts

```diff
@@ -37,7 +38,7 @@ export default Rule.Atomic.of<Page, Element>({
       expectations(target) {
         return {
           1: expectation(
-            hasAccessibleName(device, not(isEmpty))(target),
+            hasAccessibleName(device, Name.hasValue(not(isEmpty)))(target),
```

Comment:
Given that there are quite a few rules checking for non-empty accessible name, it's probably worth factoring this out in an helper function.

Reply:
Agreed 👍